com.microsoft.Edge.yaml

```diff
@@ -0,0 +1,155 @@
+app-id: com.microsoft.Edge
+# desktop-file-name-prefix: '(Testing) '
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: edge
+separate-locales: false
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - '--device=all'
+  - '--env=GTK_PATH=/app/lib/gtkmodules'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=cups'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
```

Comment:
this should be 'fallback-x11' since there is the wayland socket.

Reply:
It only runs in Wayland if `-enable-features=UseOzonePlatform -ozone-platform=wayland`. Otherwise it won't. I'm not sure if it's needed for this.

Reply:
fallback-x11 would break it, since Wayland isn't available by default.

Reply:
ack.